website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -217,19 +217,19 @@ The following arguments are supported:
 
 `network_profile` supports the following:
 
-* `network_plugin` - (Required) Network plugin to use for networking. Currently supported values are 'azure' and 'kubenet'. Changing this forces a new resource to be created.
+* `network_plugin` - (Required) Network plugin to use for networking. Currently supported values are `azure` and `kubenet`. Changing this forces a new resource to be created.
 
 -> **NOTE:** When `network_plugin` is set to `azure` - the `vnet_subnet_id` field in the `agent_pool_profile` block must be set.
 
-* `service_cidr` - (Optional) The Network Range used by the Kubernetes service. This is required when `network_plugin` is set to `kubenet`. Changing this forces a new resource to be created.
+* `service_cidr` - (Optional) The Network Range used by the Kubernetes service. Changing this forces a new resource to be created.
```

Comment:
we should add in "This field can only be set when `network_plugin` is set to `kubenet`"

Reply:
This field can only bet set with other 2 fields.

---
In reply to: [207462162](https://github.com/terraform-providers/terraform-provider-azurerm/pull/1715#discussion_r207462162) [](ancestors = 207462162)